test_client.py

```diff
@@ -0,0 +1,123 @@
+import datetime
```

Comment:
add license header 

"""
This program is free software: you can redistribute it under the terms
of the GNU General Public License, v. 3.0. If a copy of the GNU General
Public License was not distributed with this file, see <https://www.gnu.org/licenses/>.
"""